.github/workflows/circuits_e2e.yaml

```diff
@@ -34,7 +34,20 @@ jobs:
           toolchain: nightly-2024-05-22
 
       - name: Compile Circuit
-        run: nargo compile --workspace --deny-warnings
+        run: |
+          nargo compile --package get_header --deny-warnings
+          nargo compile --package get_account --deny-warnings
+          nargo compile --package get_storage --deny-warnings
+          nargo compile --package get_storage_recursive --deny-warnings
+          nargo compile --package get_receipt --deny-warnings
+          nargo compile --package get_transaction --deny-warnings
+          nargo compile --package get_log --deny-warnings
+          nargo compile --package is_dao_worthy --deny-warnings
+          nargo compile --package is_ape_owner --deny-warnings
+
+          # We cannot use the `--deny-warnings` option in `is_dao_worthy_recursive` because the `verify_proof` method generates warning 
```

Comment:
```suggestion
          # We cannot use the `--deny-warnings` option in `is_dao_worthy_recursive` because the `verify_proof` method generates warnings 
```